proto/sources.proto

```diff
@@ -40,6 +40,7 @@ enum SourceType {
   SOURCE_TYPE_SYSLOG = 25;
   SOURCE_TYPE_PUBLIC_EVENT_MONITORING = 26;
   SOURCE_TYPE_SLACK_REALTIME = 27;
+  SOURCE_TYPE_SHAREPOINT = 29;
```

Comment:
Should this be 28?

Reply:
28 is already taken

https://github.com/trufflesecurity/trufflehog/blob/f107e1b49792afcc7c5fe5f034814388fe544ed3/proto/sources.proto#L43